cloudera-integration/csd/descriptor/service.sdl

```diff
@@ -499,6 +499,14 @@
           "default": "",
           "configurableInWizard": "true"
         },
+        {
+          "name": "archway.ldap.ignoreSslCert",
```

Comment:
Nit: We should stick with uppercase acronyms `archway.ldap.ignoreSSLCert`, in the variable names as well